share-coordinator/src/test/java/org/apache/kafka/coordinator/share/ShareCoordinatorServiceTest.java

```diff
@@ -1999,7 +1999,7 @@ public void testShareStateTopicConfigs() {
         List<String> propNames = List.of(
             TopicConfig.CLEANUP_POLICY_CONFIG,
             TopicConfig.COMPRESSION_TYPE_CONFIG,
-            TopicConfig.SEGMENT_BYTES_CONFIG,
+            "internal.segment.bytes",
```

Comment:
Hmm, this doesn't look right. service.shareGroupStateTopicConfigs() doesn't include this internal config.